eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -109,7 +110,10 @@ jobs:
         inputs:
           mavenPomFile: pom.xml
           ${{ insert }}: ${{ parameters.TestStepMavenInputs }}
-        env: ${{ parameters.EnvVars }}
+        env:
+          AZURE_TEST_MODE: ${{ parameters.EnvVars }}
+          ${{ each var in parameters.EnvVars }}:
```

Comment:
This won't handle AZURE_TEST_MODE being defined twice. I think you'd need to do it in a variable declaration like:

```
parameters:
  - name: TestMode
     type: string
     default: ""
variables:
  AZURE_TEST_MODE: $[ coalesce('${{ parameters.TestMode }}', 'LIVE') ]
```

Reply:
Should this change be made in the .NET configuration where I copied this from?

Reply:
@benbp I don't think your method would prevent that either.  I personally don't think we need to worry too much about this as if someone overrrides it via env variable it is a error and we will eventually fix it.